tools/deploy-to-master-stable-branch.sh

```diff
@@ -65,6 +65,10 @@ echo "Done!"
 echo "Purging paths included in .svnignore"
 # check .svnignore
 for file in $( cat "$JETPACK_GIT_DIR/.svnignore" 2>/dev/null ); do
+	# We want to commit changes to to-test.md as well.
+	if [ $file == "$JETPACK_GIT_DIR/to-test.md" ]; then
```

Comment:
Sorry, actually this need to be changed to exclude the variable path.  Should just be `$file == "to-test.md"`